worlds/kh1/Options.py

```diff
@@ -0,0 +1,393 @@
+    """
+    display_name = "Keyblades Unlock Chests"
+
+class InteractInBattle(Toggle):
+    """
+    Allow Sora to talk, examine, and open chests in battle
+    """
+    display_name = "Interact in Battle"
+
+class AdvancedLogic(Toggle):
+    """
+    If on, logic may expect you to do advanced skips like combo master, dumbo, and other unusual methods for reaching locations.
+    """
+    display_name = "Advanced Logic"
+
+class ExtraSharedAbilities(Toggle):
+    """
+    If on, adds extra shared abilities to the pool.  These can stack, so multiple high jumps make you jump higher and multiple glides make you superglide faster.
+    """
+    display_name = "Extra Shared Abilities"
+
+class EXPZeroInPool(Toggle):
+    """
+    If on, adds EXP Zero ability to the item pool.  This is redundant if you are planning on playing on Proud.
+    """
+    display_name = "EXP Zero in Pool"
+
+class VanillaEmblemPieces(DefaultOnToggle):
+    """
+    If on, the Hollow Bastion emblem pieces are in their vanilla locations.
+    """
+    display_name = "Vanilla Emblem Pieces"
+
+class JunkInMissableLocations(DefaultOnToggle):
+    """
+    If on, will place Elixirs in missable locations.  If turned off, there is a chance you will need to play the game multiple times to get the missed items.
+    Locations include:
+        Leon Gift
+        Aerith Gift
+        Clock Tower Doors
+        World Terminus Hollow Bastion Chest
+    """
+    display_name = "Junk in Missable Locations"
+
+@dataclass
+class KH1Options(PerGameCommonOptions):
+    goal: Goal
+    end_of_the_world_unlock: EndoftheWorldUnlock
+    final_rest_door: FinalRestDoor
+    required_reports_eotw: RequiredReportsEotW
+    required_reports_door: RequiredReportsDoor
+    reports_in_pool: ReportsInPool
+    super_bosses: SuperBosses
+    atlantica: Atlantica
+    hundred_acre_wood: HundredAcreWood
+    cups: Cups
+    puppies: Puppies
+    keyblades_unlock_chests: KeybladesUnlockChests
+    interact_in_battle: InteractInBattle
+    exp_multiplier: EXPMultiplier
+    advanced_logic: AdvancedLogic
+    extra_shared_abilities: ExtraSharedAbilities
+    exp_zero_in_pool: EXPZeroInPool
+    vanilla_emblem_pieces: VanillaEmblemPieces
+    junk_in_missable_locations: JunkInMissableLocations
+    donald_death_link: DonaldDeathLink
+    goofy_death_link: GoofyDeathLink
+    randomize_keyblade_stats: RandomizeKeybladeStats
+    bad_starting_weapons: BadStartingWeapons
+    keyblade_max_str: KeybladeMaxStrength
+    keyblade_min_str: KeybladeMinStrength
+    keyblade_max_mp: KeybladeMaxMP
+    keyblade_min_mp: KeybladeMinMP
+    level_checks: LevelChecks
+    force_stats_on_levels: ForceStatsOnLevels
+    strength_increase: StrengthIncrease
+    defense_increase: DefenseIncrease
+    hp_increase: HPIncrease
+    ap_increase: APIncrease
+    mp_increase: MPIncrease
+    accessory_slot_increase: AccessorySlotIncrease
+    item_slot_increase: ItemSlotIncrease
+    start_inventory_pool: StartInventoryPool
```

Comment:
```suggestion
    start_inventory_from_pool: StartInventoryPool
```
Per conversation in discord earlier, just putting this here as a reminder since it won't work until this change is made